src/main/java/at/ac/tuwien/kr/alpha/Predicate.java

```diff
@@ -0,0 +1,4 @@
+package at.ac.tuwien.kr.alpha;
+
+public @interface Predicate {
```

Comment:
This is a marker, to be used for discovery of external predicates via annotations. I should have added a comment, sorry.

In essence, it enables code like

```java
@at.ac.tuwien.kr.alpha.Predicate
public static boolean coolPerson(String nameOfPerson) { ... }
```

to be discovered. The interface is currently empty, because we're not using any options, it just marks methods to be registered for now.